pkg/kubehunter/scanner.go

```diff
@@ -84,7 +84,13 @@ func (s *Scanner) Scan(ctx context.Context) (report starboardv1alpha1.KubeHunter
 	}()
 
 	// 4. Parse the KubeHuberOutput from the logs Reader
-	report, err = OutputFrom(logsReader)
+	version, err := starboard.GetVersionFromImageRef(s.config.GetImageRef(starboard.KubeHunterImageRef))
+	if err != nil {
+		err = fmt.Errorf("getting version from image ref: %w", err)
```

Comment:
I think we should fails fast instead of implementing some kind of fallback logic here. If you return error someone will have a chance to fix the value in the ConfigMap.